dist/wbstack/src/Settings/LocalSettings.php

```diff
@@ -44,6 +44,16 @@
     $wgShowExceptionDetails = true;
 }
 
+// profiling
+// requires docker image to be built with dependencies
+if( isset ( $_SERVER['HTTP_X_WBSTACK_MW_PROFILING'] ) ) {
+    $wgProfiler['class'] = 'ProfilerXhprof';
+    $wgProfiler['output'] = [ 'ProfilerOutputText', 'ProfilerOutputDump' ];
```

Comment:
This output format really sucks though.

Ideally this could be hooked up to xhgui for the local minikube cluster maybe? Still unclear to me how to do that.

Reply:
I think we could still move forward with this PR and have a followup to make it more usable